rio_cogeo/cogeo.py

```diff
@@ -55,8 +56,9 @@ def cog_translate(
             indexes = indexes if indexes else src.indexes
             meta = src.meta
             meta["count"] = len(indexes)
-            meta.pop("nodata", None)
-            meta.pop("alpha", None)
+            if addmask:
+                meta.pop("nodata", None)
+                meta.pop("alpha", None)
```

Comment:
When we don't add mask., we keep the src alpha or nodata value. But if a `nodata` or `alpha` value are passed via options (L20 & L21) , those are not copied in the dst profile